Formula/x/xfig.rb

```diff
@@ -21,18 +20,29 @@ class Xfig < Formula
     sha256 x86_64_linux:   "eb8139430792390d9fb6e0657d3a45081fe11f1aa74092a431fef71f614daf1b"
   end
 
+  depends_on "gnu-sed" => :build
   depends_on "fig2dev"
+  depends_on "freetype"
   depends_on "ghostscript"
   depends_on "jpeg-turbo"
   depends_on "libpng"
   depends_on "libtiff"
   depends_on "libx11"
   depends_on "libxaw3d"
+  depends_on "libxft"
   depends_on "libxi"
   depends_on "libxpm"
   depends_on "libxt"
 
   def install
+    # Use GNU sed on macOS to avoid this build failure:
+    # `sed: 1: " /^[ \t]*\(!\|$\)/ d; s ...": bad flag in substitute command: '}'`
+    ENV.prepend_path "PATH", Formula["gnu-sed"].libexec/"gnubin" if OS.mac?
+
```

Comment:
Then we can restrict the `gnu-sed` build dep to `on_macos`, right?

Reply:
good catch, updated 👍